client/src/commands/run.ts

```diff
@@ -222,6 +226,9 @@ async function _runTask(
   );
   session.onSessionLogFn = appendSessionLogFn;
 
+  const fileName = codeDoc.getUri().split("/").pop().split(".")[0];
```

Comment:
```suggestion
  const fileName = basename(codeDoc.getUri(), extname(codeDoc.getUri()));
```

Reply:
We can rely on `basename` and `extname` from `path` to handle this for us

Reply:
Cool. Done.